### PR TITLE
Added Sparked Host

### PR DESCRIPTION
**Pull request by hwalker928**
Added presence support for Sparked Host websites.

https://ss.harrydev.me/i/9o11idchaw.png
https://ss.harrydev.me/i/riz3o2pb76.png
https://ss.harrydev.me/i/xqtvbeaa5b.png

### CODE REVIEW

File: websites/S/SparkedHost/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "altar57.sparkedhost.us",
+      "web-01.sparkedhost.us"
+    ],
+    "description": {
+      "en": "Sparking your ideas to life by providing the most powerful, most reliable, and highest quality hosting at the most affordable price since 2017."
+    },
+    "service": "SparkedHost",
+    "version": "1.0.0",
+    "logo": "https://sparkedhost.com/assets/img/icon.png",
+    "thumbnail": "https://ss.harrydev.me/i/mgck2axa40.png",
```
Comment (Bas950) [inline websites/S/SparkedHost/dist/metadata.json:25]: upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

File: websites/S/Sparked Host/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "harrydev",
```
Comment (ririxidev) [inline websites/S/Sparked Host/dist/metadata.json:4]: i'm 100% sure that it isn't your discord name 😉

**Review by ririxidev — CHANGES_REQUESTED** [review]
please next time don't use this code style bruh it looks ugly

File: websites/S/Sparked Host/dist/metadata.json (hunk 2 above)
Comment (hwalker928) [reply websites/S/Sparked Host/dist/metadata.json]: Fixed

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**